webapp/src/blocks/card.ts

```diff
@@ -7,6 +7,7 @@ import {IBlock, MutableBlock} from './block'
 interface Card extends IBlock {
     readonly icon: string
     readonly isTemplate: boolean
+    readonly isDefaultTemplate: boolean
```

Comment:
Thanks @grsky360 for your PR!

Instead of adding a flag to the card, I think it's better to add a "defaultTemplateID" to the BlockView object, and store the card template's block ID there. The benefits are:
* Updating would be a single action
* We can store a different default per view

Reply:
Updated. Please review it.